packages/devtools_app/lib/src/screens/performance/panes/timeline_events/perfetto/_perfetto_controller_web.dart

```diff
@@ -42,6 +42,7 @@ class PerfettoController extends DisposableController
   void init() {
     _perfettoReady = Completer();
     _perfettoIFrame = html.IFrameElement()
+      // ignore: unsafe_html
```

Comment:
comment that this is safe because we have built the perfetto URL up ourself not received it from the user.